game/main.py

```diff
@@ -114,13 +109,15 @@ def next_level(level, levels, player):
     if keys_pressed[pygame.K_RIGHT] or keys_pressed[pygame.K_d]:
         player.move(player.move_speed, 0)
     if keys_pressed[pygame.K_DOWN] or keys_pressed[pygame.K_s]:
-        pass  # Now that we have platforms, there's no reason to make the player move down.
+        # Now that we have platforms, there's no reason to make the player move down.
+        pass
 
     if keys_pressed[pygame.K_SPACE]:
         player.create_new_bullet(level)
 
     # Mouse events
-    mouse_pos = pygame.mouse.get_pos()  # Get position of mouse as a tuple representing the
+    # Get position of mouse as a tuple representing the
+    mouse_pos = pygame.mouse.get_pos()
     # (x, y) coordinate
```

Comment:
Code formatting makes this weird.
```suggestion
    # Get position of mouse as a tuple representing the (x, y) coordinate.
    mouse_pos = pygame.mouse.get_pos()

```